.github/workflows/_rocm-test.yml

```diff
@@ -71,6 +71,7 @@ jobs:
         with:
           role-to-assume: arn:aws:iam::308535385114:role/gha_workflow_s3_and_ecr_read_only
           aws-region: us-east-1
+          role-duration-seconds: 18000
```

Comment:
This might not work as expected though because the max duration is set to 2 hours in https://github.com/pytorch-labs/pytorch-gha-infra/blob/main/runners/gha_roles.tf#L948 and https://github.com/aws-actions/configure-aws-credentials?tab=readme-ov-file#credential-lifetime says that it could not be more than that.

Reply:
This PR should fix the issue, thanks for the headsup https://github.com/pytorch-labs/pytorch-gha-infra/pull/340